### ISSUE TITLE
 Include to Red Hat Cloud services into shortcuts

### ISSUE THREAD

**Issue by PanSpagetka** [RedHatCloudForms/cfme-cloud_services#30]
In My Settings -> Visual (at the bottom of the screen) user can customize what page will show after login. Should we also include Red Hat Cloud Services? It would probably meant make `miq_shortcuts.yml` also plugable.

@h-kataria @Fryguy @martinpovolny What do you think?

**Comment by h-kataria** [comment]
@PanSpagetka yes, i think we should look into making `miq_shortcuts.yml` plugable, that will help with moving v2v shortcuts out of the yaml as well and any such future shortcuts. @GregP has been looking into a BZ to add missing shortcuts, should he wait and not add Red Hat Cloud Services shortcuts into the yaml?
@Fryguy @martinpovolny what are your thoughts, i am not sure how much amount of work that is.

**Comment by GregP** [comment]
The PR is done and ready for review, as of Friday: https://github.com/ManageIQ/manageiq/pull/19062


We do need a solution of some sort, usually a lot of UI work neglects to also provide appropriate refactoring to the mix_shortcuts settings. Though I am not really sure what the solution could/should be. And with more UI changes, and features/menus added/moved/removed, I am sure we’ll be dealing with this on a continuing basis.

**Comment by agrare** [comment]
@GregP @h-kataria we can't have any reference to RedHatCloudServices in upstream manageiq as it is a downstream only gem

**Comment by GregP** [comment]
Then we should make mix_shortcuts fix in this case part of the gem, I think.

Should I delete the commit in my PR? It is separate, for a reason (one of the 2).

**Comment by martinpovolny** [comment]
> @GregP @h-kataria we can't have any reference to RedHatCloudServices in upstream manageiq as it is a downstream only gem

Then we need the `miq_shortcuts.yml` pluggable as @h-kataria wrote or get rid of the whole `MiqShortcut` idea.

Or... ... at least for now do not add Cloud Services there and simply do not allow it to be the starting page (which probably does not make much sense anyway).

**Comment by agrare** [comment]
Yeah until we have pluggable miq_shortcuts lets just not have cloud services as an option
